src/app/zap-templates/zcl/data-model/chip/joint-fabric-datastore-cluster.xml

```diff
@@ -0,0 +1,370 @@
+    <cluster code="0x0752"/>
+    <item fieldId="0" name="GroupID" type="int64u"/>
+    <item fieldId="1" name="FriendlyName" type="char_string" length="32"/>
+    <item fieldId="2" name="GroupKeySetID" type="int16u" isNullable="true" min="1" max="65534"/>
+    <item fieldId="3" name="GroupCAT" type="int16u" isNullable="true" min="1" max="65534"/>
+    <item fieldId="4" name="GroupCATVersion" type="int16u" isNullable="true" min="1" max="65534"/>
+    <item fieldId="5" name="GroupPermission" type="DatastoreAccessControlEntryPrivilegeEnum" min="0x00" max="0x05"/>
+  </struct>
+
+  <struct name="DatastoreBindingTargetStruct">
+    <cluster code="0x0752"/>
+    <item fieldId="1" name="Node" type="node_id" optional="true"/>
+    <item fieldId="2" name="Group" type="group_id" optional="true" min="1"/>
+    <item fieldId="3" name="Endpoint" type="endpoint_no" optional="true"/>
+    <item fieldId="4" name="Cluster" type="cluster_id" optional="true"/>
+  </struct>
+
+  <struct name="DatastoreEndpointBindingEntryStruct" isFabricScoped="true">
+    <cluster code="0x0752"/>
+    <item fieldId="0" name="NodeID" type="node_id"/>
+    <item fieldId="1" name="EndpointID" type="endpoint_no"/>
+    <item fieldId="2" name="ListID" type="int16u"/>
+    <item fieldId="3" name="Binding" type="DatastoreBindingTargetStruct"/>
+    <item fieldId="4" name="StatusEntry" type="DatastoreStatusEntryStruct"/>
+  </struct>
+
+  <struct name="DatastoreEndpointGroupIDEntryStruct" isFabricScoped="true">
+    <cluster code="0x0752"/>
+    <item fieldId="0" name="NodeID" type="node_id"/>
+    <item fieldId="1" name="EndpointID" type="endpoint_no"/>
+    <item fieldId="2" name="GroupID" type="group_id"/>
+    <item fieldId="3" name="StatusEntry" type="DatastoreStatusEntryStruct"/>
+  </struct>
+
+  <struct name="DatastoreEndpointEntryStruct">
+    <cluster code="0x0752"/>
+    <item fieldId="0" name="EndpointID" type="endpoint_no"/>
+    <item fieldId="1" name="NodeID" type="node_id"/>
+    <item fieldId="2" name="FriendlyName" type="char_string" length="32"/>
+    <item fieldId="3" name="StatusEntry" type="DatastoreStatusEntryStruct"/>
+  </struct>
+
+  <enum name="DatastoreAccessControlEntryAuthModeEnum" type="enum8">
+    <cluster code="0x0752"/>
+    <item name="PASE" value="0x01"/>
+    <item name="CASE" value="0x02"/>
+    <item name="Group" value="0x03"/>
+  </enum>
+
+  <struct name="DatastoreAccessControlTargetStruct">
+    <cluster code="0x0752"/>
+    <item fieldId="0" name="Cluster" type="cluster_id" isNullable="true"/>
+    <item fieldId="1" name="Endpoint" type="endpoint_no" isNullable="true"/>
+    <item fieldId="2" name="DeviceType" type="devtype_id" isNullable="true"/>
+  </struct>
+
+  <struct name="DatastoreAccessControlEntryStruct">
+    <cluster code="0x0752"/>
+    <item fieldId="1" name="Privilege" type="DatastoreAccessControlEntryPrivilegeEnum" isFabricSensitive="true" min="0x00" max="0x05"/>
```

Comment:
Spec issue: you can't have fabric-sensitive fields in a struct that is not fabric-scoped, because how is that supposed to work?  The whole point of fabric-sensitive fields is that they are elided when read on a fabric that does not match the associated fabric of the struct, but that means the struct needs an associated fabric.

Concretely: the codegen in the SDK does NOT treat these fields as fabric-sensitive, because this is not a fabric-scoped struct.  So either the spec is wrong or the SDK is wrong here.

Reply:
DatastoreAccessControlEntryStruct (AccessControlEntryStruct) was originally set as fabric scoped, but changed in the spec as a part of #11538. We will review this again as a part of https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/11620